doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -161,10 +189,12 @@ an opaque way. Nonetheless, the types described below are used to convey an spec
   greater than, lesser than, greater or equal, lesser or equal and range query operators. Eg:
 
 ```
+{
```

Comment:
The FABRE tool should adpat to the input it has to process and not the oposite, but I think we can live with this ;)